src/App.js

```diff
@@ -1,49 +1,48 @@
 import './App.css';
 import axios from 'axios';
 import React, { useEffect, useState } from 'react';
-import Moment from 'moment';
+import NotificationsFlyout, { anchorPoints }  from "@hig/notifications-flyout";
+import Timestamp from "@hig/timestamp";
 
 function App() {
-
+  let notificationData = [];
   const [APIData, setAPIData] = useState([]);
   useEffect(() => {
     axios.get(`http://demo9540080.mockable.io/notifications`)
       .then((response) => {
         setAPIData(response.data.notifications);
       })
   }, []);
-
+  
+  for (let i = 0; i < APIData.length; i++) 
+  {
+	var notificationItem = {
+							  id: APIData[i].id,
+							  featured: true,
+							  unread: true,
+							  message: APIData[i].title,
+							  timestamp: <Timestamp timestamp= {APIData[i].created} />,
+							  content: <div>
+											<a style={{marginRight:'15px', marginTop:'-2px'}}><img width="20" src={APIData[i].thumbnail}/></a>
```

Comment:
I remember I had some problems using 'px' values for our HTML pages in the guided tours. Maybe it's better to use 'rem' instead. https://nekocalc.com/px-to-rem-converter

Reply:
Changed it in the new PR. More styling can be done once we confirm the workflow.